lxd/device/proxy.go

```diff
@@ -410,14 +410,19 @@ func (d *proxy) setupProxyProcInfo() (*proxyProcInfo, error) {
 		return nil, fmt.Errorf("Invalid binding side given. Must be \"host\" or \"guest\"")
 	}
 
+	listenAddrMode := "0755"
```

Comment:
I think the default for a socket should be 0644 not 0755.

Can you update the doc and the value?

Reply:
@stgraber sure thing, although I now notice the docs for the other file based devices; usb, gpu, unix-char, unix-block, default to 0660, would this be preferable for consistency?

Reply:
@stgraber although we should be careful not to change the defaults such that it breaks things for users currently using the proxy with no mode set.

Reply:
0660 for devices is reasonable, 0644 for sockets is reasonable. It's fine that they aren't the same.

When passing a network connection into the container, I think most will expect to have everything in the container be able to connect to it.

When passing a device however (especially for block), I don't think everyone would be happy with a nobody user being able to interact with it.

Reply:
@stgraber this has been updated to 0644.